provision/openshift/Taskfile.yaml

```diff
@@ -256,6 +256,7 @@ tasks:
         --set infinispan.remoteStore.enabled=$(cat .task/remote-store-enabled)
         --set infinispan.remoteStore.host=$(cat .task/remote-store-host)
         --set infinispan.remoteStore.password=$(cat .task/remote-store-password)
+        --set infinispan.site={{.KC_NAMESPACE_PREFIX}}keycloak
```

Comment:
this value is needed for any single site ISPN installation, otherwise it would fail with an exception around `jboss.site.name` during the Keycloak Deployment.